games-emulation/sameboy-jg/sameboy-jg-9999.ebuild

```diff
@@ -15,7 +15,7 @@ if [[ "${PV}" == *9999 ]] ; then
 else
 	SRC_URI="https://gitlab.com/jgemu/${MY_PN}/-/archive/${PV}/${MY_P}.tar.bz2"
 	S="${WORKDIR}/${MY_P}"
-	KEYWORDS="~amd64"
+	KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~x86"
```

Comment:
You are adding lot's of arches. Did you test on all of them?

Reply:
Yes, see issue https://bugs.gentoo.org/891201. Here I am only updating the live ebuilds accordingly.